MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m

```diff
@@ -6,9 +6,166 @@
 //
 
 #import "SpotifyAPIManager.h"
+#import <Parse/Parse.h>
+#import "AppDelegate.h"
+#import "MatchmakingViewController.h"
+#import <CoreData/CoreData.h>
+#import "ConnectView.h"
+#import "LoginViewController.h"
 
 @implementation SpotifyAPIManager
 
 
++ (instancetype)shared {
+    static SpotifyAPIManager *sharedManager = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        sharedManager = [[self alloc] init];
+    });
+    return sharedManager;
+}
+
+static NSString * const SpotifyClientID = @"9f9a8a428178497e8c58840c65d9f3c0";
+static NSString * const SpotifySecretID = @"085725985e87480fab42de4970086a54";
```

Comment:
Reminder that this is a public repository, so by committing your secret here anyone can access your spotify account. Probably not a huge deal (depending on what that access allows them to do) but wanted to call it out. Generally the strategy is to put this type of thing in a special Info.plist that is in gitignore (so it never ends up on Github).